### PR TITLE
Depend on WPE WebKit 2.28, libwpe+WPEBackend-fdo 1.5

### LINKED ISSUE #241: Bump required libwpe version to 1.5.0 and remove 2D axis event conditionals
There are a few places in the code that have a check similar to this:

```c
#if defined(WPE_CHECK_VERSION)
# define HAVE_2D_AXIS_EVENT WPE_CHECK_VERSION(1, 5, 0) && WEBKIT_CHECK_VERSION(2, 27, 4)
#else
# define HAVE_2D_AXIS_EVENT 0
#endif /* WPE_CHECK_VERSION */
```

Versions 1.6.x of libwpe and 2.28.x have been around for a while, so we could change the minimum requirement to those and remove the conditionals to always use 2D axis events.

### PR DESCRIPTION
**Pull request by aperezdc**
Update the minimum required versions to the following, which are all more than one year old:

- WPE WebKit 2.27.3, the oldest development version of the 2.28 series   which already has most of the features used by Cog.
- libwpe and WPEBackend-fdo 1.5.0, with the same reasoning.

This allows removing a bunch of conditional code.

Fixes #241

### CODE REVIEW

**Comment by aperezdc** [comment]
I think that this could tentatively be part of the 0.10 release. If anybody else thinks it's late for this in the development cycle I am open to merge this after branching for 0.10—which should be soon anyway :smile_cat:

**Comment by aperezdc** [comment]
This turned out to be a bit big of a change, so in a chat with @zdobersek we have agreed to merge it _after_ the branch for the stable 0.10 releases is made.

**Comment by aperezdc** [comment]
Ping reviewers 🏓 — the `cog-0.10` release branch has been created, so we could not merge this into main development branch.

File: CMakeLists.txt
Comment (philn) [inline CMakeLists.txt:118]: s/2.27.3/2.28.0 ?

**Review by philn — CHANGES_REQUESTED** [review]
LGTM, just a minor comment

File: CMakeLists.txt
Comment (philn) [reply CMakeLists.txt:118]: I guess I get the initial reasoning though, but I wouldn't really expect anyone to use pre-releases for their products.

Comment (aperezdc) [reply CMakeLists.txt:118]: Development release 2.27.3 happens to be the very first one that supports all the features that this PR removes the conditional compilation guards for. I don't see a problem in leaving it like that, but also there is no downside using 2.28.0, so I'll update the PR with that 😄

Comment (aperezdc) [reply CMakeLists.txt:118]: Also, I'll change it things to use `wpe-1.0>=1.6.0` and `wpebackend-fdo-1.6.0` while at it.

Comment (aperezdc) [reply CMakeLists.txt]: Done!

**Review by philn — APPROVED** [review]

**Comment by aperezdc** [comment]
Thanks @philn 😃

**Pull request merged**
